### DISCUSSION TITLE
Error with name that contains an uppercase letter

### DISCUSSION THREAD

**Discussion by Yusseiin** [alexandrainst/node-red-contrib-postgresql#32] [Q&A] [answered]
Hi,
thanks for the node.
I have a problem when i am trying to reach a table with a uppercase letter.
I have a table named "Gant"
![image](https://user-images.githubusercontent.com/103202144/208077726-d799cbda-5303-4da1-b820-cac8cdbd83c1.png)
When i try to select from it i receive an error back:
"error: relation "public.gant" does not exist"
![image](https://user-images.githubusercontent.com/103202144/208077635-ecfb2c50-dafe-4107-b188-216e5142539c.png)
I think the problem is that the node is searching for "gant" and not for "Gant", if i rename the table from PostgreSQL in "gant" everything work.

Edit:
The same error is recurring with the Columns name
![image](https://user-images.githubusercontent.com/103202144/208079857-d69bf34f-49ff-495e-862a-b6f1f3bd30dc.png)

![image](https://user-images.githubusercontent.com/103202144/208079836-259e40f2-ff7e-4081-baa8-2152dba2ccaf.png)

**Comment by Alkarex** [comment, marked as answer]
Hello,
PostgreSQL is case sensitive when it comes to table names, if they have been created in a case-sensitive way.
The behaviour is not due to this Node-RED node, which passes the query directly to the database.
To address a table name or column name that have uppercase letters, you need to use double quotes around them:

```sql
SELECT * FROM "Gant" LIMIT 2;
```

See https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS

**Reply by Yusseiin** [reply]
It still has some problem.
I'am missing something?
![image](https://user-images.githubusercontent.com/103202144/208134855-b5ff1f01-0b0d-412c-8d55-4505e9f11266.png)
![image](https://user-images.githubusercontent.com/103202144/208134956-9a6f395d-bd6a-4461-87ea-94e21b48317b.png)

**Comment by Alkarex** [comment]
Can you try the same request in pgAdmin and show the results?

**Comment by Yusseiin** [comment]
Actually i got the same error from pgAdmin
![image](https://user-images.githubusercontent.com/103202144/208136224-e300de9b-f052-4e3c-91ca-72fd07fbdcde.png)
I have managed to get working escaping only the table and not public
![image](https://user-images.githubusercontent.com/103202144/208136554-05906d11-b048-4c42-99c4-48023fc628ca.png)
Thanks for the help
